src/openads/application/service/DisplayPositionUseCase.js

```diff
@@ -0,0 +1,30 @@
+import {POSITION_VISIBLE} from '../../domain/position/positionStatus'
+
+export default class DisplayPositionUseCase {
+  /**
+   * @constructor
+   * @param {PositionRepository} positionRepository
+   */
+  constructor ({positionRepository}) {
+    this._positionRepository = positionRepository
+  }
+
+  /**
+   * Displays a position in the page
+   * @param {string} position id
+   * @return {Promise<Position>}
+   */
+  displayPosition ({id} = {}) {
+    return this._positionRepository.find({id})
+      .then(this._resolveOptionalPosition)
+      .then(foundPosition => foundPosition.changeStatus({newStatus: POSITION_VISIBLE}))
+      .then(modifiedPosition => this._positionRepository.save({position: modifiedPosition}))
+  }
+
+  _resolveOptionalPosition (optionalPosition) {
+    if (optionalPosition === null) {
+      throw new Error('PositionNotFound')
```

Comment:
For consistency, please, talk and reach an agreement with @victorcaldentey about the Domain Errors, to not throw an Error with a message.

Reply:
Sure, we will use custom Errors with fixed messages.